src/services/transaction.rs

```diff
@@ -0,0 +1,14 @@
+use uuid::Uuid;
+
+use crate::database;
+use crate::models::transaction::{self, Transaction};
+
+fn create_transaction(t: Transaction) -> Transaction {
+    let mut conn = database::establish_connection();
+    transaction::create_transaction(&mut conn, t)
+}
+
+pub fn list_user_transactions(user_id: Uuid) -> Vec<Transaction> {
+    let mut conn = database::establish_connection();
```

Comment:
Please, add a todo/issue about using the same connection/pool through multiple requests.